mosquitto/aclfile

```diff
@@ -0,0 +1,2 @@
+# Only allow messages to be read and written within topics that match the username
+pattern readwrite %u/#
```

Comment:
This is what restricts users to their own messages.  It is used in combination with the `acl_file /mosquitto/config/aclfile` entry in `mosquitto/mosquitto.conf`.